docs/core/config/auth/databases/ldap.md

```diff
@@ -220,12 +235,12 @@ reconnecting to the "primary" server.
 
 You can enable TLS in two alternative ways:
 
-* Connect to ldaps port (636) by using "ldaps" protocol, e.g. `uris =
+* Connect to ldaps port (636) by using "ldaps" protocol, e.g. `ldap_uris =
   ldaps://secure.domain.org`
-* Connect to ldap port (389) and use STARTTLS command. Use `tls=yes` to
+* Connect to ldap port (389) and use STARTTLS command. Use `ssl=yes` to
   enable this.
 
-See the `tls_*` settings in `dovecot-ldap-example.conf` for how to
+See the `ssl_*` settings in `dovecot-ldap-example.conf` for how to
 configure TLS.
```

Comment:
This probably was supposed to be `example-config/dovecot-ldap.conf.ext` in the 2.3 code, but in config-rewrite this won't exist. Need to rewrite in some way.